docs/DeveloperGuide.md

```diff
@@ -247,3 +247,58 @@ Main Success Scenario (MSS) defines the optimal outcome of our commands, i.e. in
 
 ---
 
+## Appendix: Instructions for manual testing
```

Comment:
![image](https://user-images.githubusercontent.com/60131677/114389876-22877800-9bc8-11eb-9a12-c67145ff1e31.png)

Lol guess we need to write testing for 
* Entry: Add, Filter tags, Find, List day/week, Free, Edit, clear overdue 
* Contacts: Add (since we change address), filter, edit

Reply:
rip will do

Reply:
I can write the testing for entry?